subrpcserver_config.go

```diff
@@ -59,6 +61,10 @@ type subRPCServerConfigs struct {
 	// as a gRPC service.
 	InvoicesRPC *invoicesrpc.Config `group:"invoicesrpc" namespace:"invoicesrpc"`
 
+	// NeutrinoKitRPC is a sub-RPC server that exposes functionality allowing
+	// a client to interact with a running neutrino node.
+	NeutrinoKitRPC *neutrinorpc.Config `group:"neutrinorpc" namespace:"neutrinorpc"`
```

Comment:
Config doesn't exist in this commit yet? We try to get each commit to compile, so that it's easier to `bisect` when bughunting. Should be as simple as reordering commits.

Reply:
@carlaKC Thank you. I'll make sure they compile separately.